mobile/src/bundles/employer/helpers/transform-candidate-filter-form-to-query/transform-candidate-filter-form-to-query.ts

```diff
@@ -0,0 +1,35 @@
+import { type EmployeesFiltersForm } from '~/bundles/employer/types/types';
+
+const transformCandidateFilterFormToQuery = (
+    formData: EmployeesFiltersForm,
+): string => {
+    const {
+        employmentType,
+        englishLevel,
+        searchActiveCandidatesOnly,
+        ...multiSelectedData
+    } = formData;
+    let result = `?searchActiveCandidatesOnly=${searchActiveCandidatesOnly}`;
+    if (englishLevel?.length) {
+        for (const value of englishLevel) {
+            result += `&englishLevel=${value}`;
+        }
+    }
+    if (employmentType?.length) {
+        for (const value of employmentType) {
+            result += `&employmentType=${value}`;
+        }
+    }
+
+    for (const [key, value] of Object.entries(multiSelectedData)) {
```

Comment:
thinking about another helper to simplify thoose for of, but dont knwo is it a good idea to create helper for helper?)